core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -241,10 +245,55 @@ public VirtualHostBuilder tls(KeyManagerFactory keyManagerFactory,
         requireNonNull(keyManagerFactory, "keyManagerFactory");
         requireNonNull(tlsCustomizer, "tlsCustomizer");
 
-        tls(buildSslContext(() -> SslContextBuilder.forServer(keyManagerFactory), tlsCustomizer));
+        final SslContext sslContext = buildSslContext(() -> SslContextBuilder.forServer(keyManagerFactory),
+                                                      tlsCustomizer);
+        try {
+            validateKeyStoreNullPassword(sslContext);
+        } catch (Exception e) {
+            throw new RuntimeException("failed to validate SSL/TLS configuration: " + e.toString(), e);
+        }
```

Comment:
Once we merge this into `validateSslContext()`, we do not need to validate here but only in `tls(SslContext)`.

Reply:
Removed.